### ISSUE TITLE
Cypress loads blank page on attempt to open test app url in Chromium-based browsers

### ISSUE THREAD

**Issue by hraichonaky** [cypress-io/cypress#24418]
### Current behavior

On attempt to open simple app url in primitive Cypress (10.11) test (via Chrome browser) - blank page is loaded.
Use following URL http://power-inspiration-1088-dev-ed.scratch.my.site.com/ebikes for reproduction
In Chrome
![image](https://user-images.githubusercontent.com/116178405/198154272-5351f8d5-41d8-4954-b634-2141d5a8ec5f.png)

In Firefox
![image](https://user-images.githubusercontent.com/116178405/198154408-d51d3bb8-dc67-49a6-ba74-ba3e07a01721.png)

### Desired behavior

Cypress should load page content instead of blank page. 
When user opens the app manually ( i.e in non-cypress triggered) in Chrome  - content is loaded completely.
Problem is reproducible in all Chromium-based browsers (Chrome/Edge/Electron). 
In Firefox problem is NOT reproducible in Cypress

### Test code to reproduce

cypress.config.js
```
const { defineConfig } = require("cypress");

module.exports = defineConfig({
   chromeWebSecurity: false,
  e2e: {
    setupNodeEvents(on, config) {
      // implement node event listeners here
    },
    "viewportWidth":1980,
    "viewportHeight":1020,
    "screenshotsFolder": "mochawesome-report/assets",
    "videosFolder": "mochawesome-report/assets",
     baseUrl: 'http://power-inspiration-1088-dev-ed.scratch.my.site.com/ebikes',
  },
});
```
login.cy.js

```
describe('Sales Force First Test', () => {
  it('Open page', () => {
    cy.visit('/')
  })
})
```

### Cypress Version

10.11.0

### Node version

16.18.0

### Operating System

12.6

**Comment by hraichonaky** [comment]
[debug.log](https://github.com/cypress-io/cypress/files/9874328/debug.log)

**Comment by liambutler** [comment]
I also saw this on the https version of the above url

**Comment by ricardostaudt** [comment]
Just to give a little more context to the problem: this is a Salesforce page and even though it doesn't require authentication it may have something to do with the salesforce authentication process, I've had similar problems running Salesforce tests with Selenium in environments that had security features/monitors enabled. My guess is that the way Cypress captures traffic interferes with Salesforce authentication, causing the page to not load

**Comment by mschile** [comment]
Thanks for the update. I'm going to try and take a look at this today to see if I can determine what's going on.

**Comment by mschile** [comment]
@hraichonaky, fyi, it looks like https://power-inspiration-1088-dev-ed.scratch.my.site.com/ebikes/s/ is currently down.

![Image](https://user-images.githubusercontent.com/2002044/199559685-2e45d696-516a-4760-9797-f073d8d20646.png)

**Comment by hraichonaky** [comment]
> @hraichonaky, fyi, it looks like https://power-inspiration-1088-dev-ed.scratch.my.site.com/ebikes/s/ is currently down.
> 
> ![Image](https://user-images.githubusercontent.com/2002044/199559685-2e45d696-516a-4760-9797-f073d8d20646.png)

@mschile 
Seems that Salesforce playground environments are temporary (flushed in few days). Just deployed a new on  - please use that https://customer-force-8778-dev-ed.my.site.com/ebikes/s/

**Comment by mschile** [comment]
This issue appears to be related to #24290. We hope to investigate it soon to find a solution.
<img width="1492" alt="Screen Shot 2022-11-04 at 8 02 11 AM" src="https://user-images.githubusercontent.com/2002044/199991154-341046e1-f13e-4361-802c-b7e39af2ab29.png">

**Comment by ricardostaudt** [comment]
More information that might be helpful: I tried to access this same page with playwright's webkit browser and got this error:

```
page.goto: Stream error in the HTTP/2 framing layer
    =========================== logs ===========================
    navigating to "https://customer-force-8778-dev-ed.my.site.com/ebikes/s/", waiting until "load"
    ============================================================
```

This error doesn't happen with other browsers with playwright. By googling this error message it seems to be related to HTTP/2 authentication and in some cases people have had success after disabling HTTP/2 when such option is available (curl has it).

**Comment by alexanderg1982** [comment]
Hi, this is similar to a ticket that I opened #24303 that was merged to #24290

**Comment by AtofStryker** [comment]
Hey all. We appreciate your patience on getting this issue resolved. This will be fixed in Cypress `12.4.0` with a new experimental flag mentioned in https://github.com/cypress-io/cypress/issues/24290#issuecomment-1376214221. In this case, you will likely need to configure the flag to look something like this to work: 

```js

const { defineConfig } = require("cypress");

module.exports = defineConfig({
  e2e: {
    baseUrl: "http://power-inspiration-1088-dev-ed.scratch.my.site.com/ebikes",
    experimentalModifyObstructiveThirdPartyCode: true,
    experimentalSkipDomainInjection: ['*.salesforce.com', '*.force.com', '*.scratch.my.site.com']
  },
});

```
